bootstrap/pkg/utils/k8utils.go

```diff
@@ -318,6 +319,52 @@ func (a *Apply) namespace(namespace string) error {
 	return nil
 }
 
+//LabelClusterRole labels ClusterRoles with labelKey: labelValue pair
+func (a *Apply) LabelClusterRole(clusterRoleName string, labelKey string,
```

Comment:
Since this is a very specific function needed in a certain place only, I believe it would be better to contain it within that file (kustomize.go)?
You can create a k8s client there as well.

Reply:
It makes sense - I'll make this change.